dbfcreate.c

```diff
@@ -65,6 +65,32 @@ int main(int argc, char **argv)
             }
             i += 3;
         }
+        else if (i < argc - 1 && strcmp(argv[i], "-d") == 0)
```

Comment:
"-d" and "-s" need to be documented in the usage at line 25-26

Reply:
Done.

I suppose they should be documented in the man pages too, but didn't find them in the repository. Also, my English is not good.